lib/datadog/appsec/contrib/graphql/appsec_trace.rb

```diff
@@ -16,16 +16,10 @@ def execute_multiplex(multiplex:)
 
             gateway_multiplex = Gateway::Multiplex.new(multiplex)
 
-            multiplex_return, multiplex_response = Instrumentation.gateway.push('graphql.multiplex', gateway_multiplex) do
+            multiplex_return, = Instrumentation.gateway.push('graphql.multiplex', gateway_multiplex) do
```

Comment:
It's a matter of preference, but it's bugging me 😆 Maybe we can go more explicit?

```suggestion
            multiplex_return, _ = Instrumentation.gateway.push('graphql.multiplex', gateway_multiplex) do
```

Reply:
it bugs me as well, but this is due to rubocop :( I will check if we could add disable this